rero_ils/modules/documents/serializers/dc.py

```diff
@@ -0,0 +1,175 @@
+# -*- coding: utf-8 -*-
+#
+# RERO ILS
+# Copyright (C) 2022 RERO
+# Copyright (C) 2022 UCLouvain
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, version 3 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""DublinCore Document serialization."""
+
+from dcxml import simpledc
+from flask import request
+from invenio_records_rest.serializers.dc import \
+    DublinCoreSerializer as BaseDublinCoreSerializer
+from lxml import etree
+from lxml.builder import ElementMaker
+
+from rero_ils.modules.documents.api import Document
+from rero_ils.modules.documents.dojson.contrib.jsontodc import dublincore
+from rero_ils.modules.documents.utils import create_contributions
+
+DEFAULT_LANGUAGE = 'en'
```

Comment:
Is this should be more general? Do we need to declare this at the application level?

Reply:
ok